src/inference/src/threading/ie_cpu_streams_executor.cpp

```diff
@@ -98,7 +98,18 @@ struct CPUStreamsExecutor::Impl {
                     // wrapping around total_streams (i.e. how many streams all different core types can handle
                     // together)
                     const auto total_streams = _impl->total_streams_on_core_types.back().second;
+                    const auto hybrid_core = _impl->total_streams_on_core_types.size() > 1;
```

Comment:
The implementation inside this ctor become much more complex than before. However the logic itself is not so difficult. 
Can we have an explanation in one place (like just before Streams ctor signature) with behavior for each pining mode? For Hybrid awere in short this will be: binding streams to cores in round robin manner with following priorities: Big core, Small core, Logical big core...

Reply:
Sure. Will create following PR for pining mode explanation.